### PR TITLE
Add Basic MasteryProgress Screen, Connect to Dev Backend

### PR DESCRIPTION
**Pull request by ConnorBoulais**
This PR adds a first version of the mastery progress screen, which exists at the route `/mastery`. This screen is not finished but is functional as it is. It will likely be updated.

The PR also connects our app to the backend Dev API instead of the Prod API.  This will allow us to develop against the new APIs that were recently added to the backend, many of which we need.

After merging this PR, everyone will have to update their .env file.   Ask me and I'll send you the updated .env file content.

### CODE REVIEW

File: src/Components/MasteryProgress/MasteryIcon.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { MASTERY_TAG } from './MasteryLabel';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import StarIcon from '@material-ui/icons/Star';
+
+function MasteryIcon(props){
```
Comment (tddumont) [inline src/Components/MasteryProgress/MasteryIcon.js:5]: Good thinking to make this icon its own component!

File: src/Components/MasteryProgress/MasteryLabel.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const labelStyle = {
+    backgroundColor: getLabelColor(),
+    borderRadius: "1em",
+    width: "max-content",
+    paddingLeft: "2em",
+    paddingRight: "2em",
+    fontFamily: "\"Poppins\", sans-serif",
+  };
+
+  function getLabelText(){
```
Comment (ryanmadamba) [inline src/Components/MasteryProgress/MasteryLabel.js:28]: This is a smart way of distinguishing each of the labels

**Review by ryanmadamba — APPROVED** [review]
Looks good as always, Connor!

File: src/gqlQueries.js [hunk 3]
```diff
@@ -340,9 +340,11 @@ export const GET_ALL_TARGET_PROGRESS = gql `
         objectiveId
       	objectiveName
         tasks {
-          taskId
```
Comment (tddumont) [inline src/gqlQueries.js:343]: Makes sense to not retrieve data that isn't used.

File: src/Components/MissionsScreen.js [hunk 4]
```diff
@@ -75,7 +75,7 @@ export function MissionsScreenDisplay(data, progressData, hist, focusedMission,
               done={prog}
             />
           </div>
-          <h1 style={{"font-size": "18px", margin: "0", padding: "0", align: "center", "padding-bottom": "20px"}}>{prog}% Complete</h1>
+          <h1 style={{"font-size": "18px", margin: "0", padding: "0", align: "center", "padding-bottom": "20px"}}>{Math.round(prog)}% Complete</h1>
```
Comment (tddumont) [inline src/Components/MissionsScreen.js:78]: Thanks for catching this!

File: src/Components/MasteryProgress/TargetDetails.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    fontSize: "16px"
+  },
+});
+
+function Row(props) {
+  const { row } = props;
+  const [open, setOpen] = React.useState(false);
+  const classes = useRowStyles();
+
+  return (
```
Comment (tddumont) [inline src/Components/MasteryProgress/TargetDetails.js:42]: Great organization!

**Review by tddumont — APPROVED** [review]
Looks great!

**Pull request merged**